src/app-layer-smb.c

```diff
@@ -28,199 +28,6 @@
 #include "app-layer-smb.h"
 #include "util-misc.h"
 
-#define MIN_REC_SIZE 32+4 // SMB hdr + nbss hdr
-
-static AppLayerResult SMBTCPParseRequest(Flow *f, void *state,
-        AppLayerParserState *pstate, const uint8_t *input, uint32_t input_len,
-        void *local_data, const uint8_t flags)
-{
-    SCLogDebug("SMBTCPParseRequest");
-    uint16_t file_flags = FileFlowToFlags(f, STREAM_TOSERVER);
-    rs_smb_setfileflags(0, state, file_flags|FILE_USE_DETECT);
-
-    if (input == NULL && input_len > 0) {
```

Comment:
I do not see this gap handling code in rust in this commit where it gets removed from C... Am I missing something ?

Reply:
It is added in the commit before this removal.